code/game/objects/structures/vehicles/carts/cargo.dm

```diff
@@ -48,7 +45,7 @@
 	if(!isturf(C.loc)) //To prevent the loading from stuff from someone's inventory, which wouldn't get handled properly.
 		return FALSE
 
-	if(C.locked_to || C.is_locking())
+	if(C.locked_to || C.is_locking() || C.anchored)
```

Comment:
quality feature you got there

Reply:
thanks I made it myself

Reply:
I think you can just remove this check entirely. The feature is now a feature, not an accidental feature.